k8s/ithc/common/xui/webapp.yaml

```diff
@@ -29,8 +29,11 @@ spec:
         LAUNCH_DARKLY_CLIENT_ID: 1
         DEBUG: xuiNode:*,-xuiNode:auth:s2s
         ISS_SERVICE: https://forgerock-am.service.core-compute-idam-ithc.internal:8443/openam/oauth2/hmcts
-        SERVICES_IDAM_OAUTH_CALLBACK_URL: /oauth2/callback
+        SERVICES_IDAM_OAUTH_CALLBACK_URL: oauth2/callback
```

Comment:
Why are you removing this /

Reply:
same setting as MC demo

Reply:
this is currently in MC demo which is working right, if we going to use the same tag, this env var will need to use the same too? @udaydenduluri33 https://github.com/hmcts/cnp-flux-config/blob/master/k8s/demo/common/xui/webapp.yaml#L33

Reply:
If using the new node lib, then it's /ouath2/callback

Reply:
we using exact same tagged image as demo, so env var should be the same right? until we get a new PR, i think thats should be correct